### ISSUE TITLE
"Activities" section no longer shows Download or Uploads

### ISSUE THREAD

**Issue by JRM02892** [microsoft/AzureStorageExplorer#7288]
### Preflight Checklist

- [X] I have installed the [latest version of Storage Explorer](https://github.com/Microsoft/AzureStorageExplorer/releases/latest).
- [X] I have checked existing resources, including the [troubleshooting guide](https://docs.microsoft.com/azure/storage/common/storage-explorer-troubleshooting) and the [release notes](https://github.com/microsoft/AzureStorageExplorer/releases).
- [X] I have searched for [similar issues](https://github.com/Microsoft/AzureStorageExplorer/issues).

### Storage Explorer Version

1.31.1

### Regression From

Whichever number was immediately before this.

### Architecture

x64

### Storage Explorer Build Number

20230822.1

### Platform

macOS

### OS Version

Ventura 13.5.1

### Bug Description

Previously, Storage Explorer would show the progress of downloads and uploads in the "Activities" section of the Screen. Since updating to the newest version, this section is perpetually blank. Even when downloading or uploading, there is no visual confirmation that the process is underway or completed.
![Screenshot 2023-09-11 at 10 09 54 AM](https://github.com/microsoft/AzureStorageExplorer/assets/144707948/a01bab8c-a187-47a8-8c5e-8468be05ba7f)

### Steps to Reproduce

1. Launch Storage Explorer
2. Download or Upload any item from File Share

### Actual Experience

Per my screenshot in the bug description, there is no visual feedback of items downloading or uploading to the file share. The activities section, which normally lists all of these details, perpetually stays blank. Attempting to "clear" completed or successful does nothing.

### Expected Experience

I expect the old functionality of feedback for uploads and downloads. Especially when moving large files, it is necessary to see percentage of completion, or written feedback of when a project fails. Currently, the only way to confirm is to manually search and compare notes, which is tedious for folders over 2GB.

### Additional Context

There are no errors that pop up. The feature just acts like it is disabled.

**Comment by JRM02892** [comment]
[2023-09-11_095536_JobHandlerProvider_large_0_84370.log](https://github.com/microsoft/AzureStorageExplorer/files/12578106/2023-09-11_095536_JobHandlerProvider_large_0_84370.log)

Let me know if this is not what you're looking for.

**Comment by MRayermannMSFT** [comment]
@JRM02892 unfortunately not quite. There should be many files in the directory where you got the file you just shared. We need you to share all of those files, preferably all collected into a single zip.

**Comment by JRM02892** [comment]
[2023-09-11_095536.zip](https://github.com/microsoft/AzureStorageExplorer/files/12578358/2023-09-11_095536.zip)

**Comment by MRayermannMSFT** [comment]
Thank you, this is very helpful. I think we'll likely reach back out for additional info/have you try things based on my initial look. We'll let you know what we need after we invesigate a bit deeper. Thanks again.

**Comment by JasonYeMSFT** [comment]
@JRM02892 Does the problem keeps reproducing even if you restart Storage Explorer? If it keeps reproducing, could you do the following thing for us to troubleshoot? Right click the Storage Explorer app and select "Show Package Contents". Then navigate in the package contents and share this file with us `Contents/Resources/app/node_modules/@storage-explorer/activity-log/dist/src/Panels/ActivityLog/index.html`. There is a mysterious error originated from this file which breaks displaying the activities in the activity log. Also, I noticed that you are running Storage Explorer from the default download location which enables Apple's AppTranslocation. It anonymizes the path of the application files and can sometimes lead to unexpected behaviors. You can try copying Storage Explorer app bundle to you `Applications` folder and run from there to see if it helps you mitigate this issue.

**Comment by craxal** [comment]
I've noticed from time to time error messages popping up that say something about a template binding error. I suspect there's a UI bug somewhere.

**Comment by craxal** [comment]
```
[2023-09-15T00:01:09.797Z] (editor:60206) <CRIT> Unhandled rejection, reason: Error: Unable to process binding "template: function(){return { name:'activity-log-template',data:$data} }"
  Message: Cannot find template with ID activity-log-template
      ...
```

**Comment by craxal** [comment]
@JRM02892 If you close and relaunch Storage Explorer, does the Activity Log ever function normally, or is it consistently nonfunctional across sessions?

**Comment by JRM02892** [comment]
@craxal low and behold, it decided to work for me this morning after a full reboot and launching from applications. Surprising as the same steps had not solved the issue previously.

**Comment by craxal** [comment]
We'll keep this open and continue investigating, as we've also encountered the issue.

**Comment by JasonYeMSFT** [comment]
I found the bug. Will fix in 1.32.0.
